ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -1650,15 +1648,13 @@ tests:
     env:
       COMPUTE_NODE_TYPE: t2a-standard-4
       E2E_RUN_TAGS: not @storage
-      FEATURE_GATES: '["ClusterAPIInstall=true"]'
-      FEATURE_SET: CustomNoUpgrade
       OCP_ARCH: arm64
       SIZE_VARIANT: compact
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-filestore-csi
-- as: gcp-ipi-disc-priv-capi-amd-arm-day0-f28-destructive
```

Comment:
The change was introduced from https://github.com/openshift/release/pull/54703, I think we can remove `capi` keyword, and `"ClusterAPIInstall=true"` from `FEATURE_GATES`, keep others.

cc @aleskandro to confirm

Reply:
Yes we can

Reply:
I can update it in https://github.com/openshift/release/pull/55382